README.md

```diff
@@ -9,12 +9,219 @@ This little command line tool enables you to conveniently pull BO4E-Schemas of a
 Additionally, it supports some features to edit those schemas which can be defined by config values.
 
 ## Features
+### Overview
 
 - Pull BO4E-Schemas of arbitrary versions
+  - references in `$ref` will be replaced by relative paths
 - Edit schemas:
   - Define required properties
   - Add additional properties (keep in mind that you should avoid this if possible)
-  - Maybe future support for basic validation of the schemas
+  - Add additional models (keep in mind that you should avoid this if possible)
+  - All features also apply to Enums
+
+## Usage
+
+This tool is a command line tool. You can use it by installing `BO4E-Schema-Tool` via
+```bash
+pip install BO4E-Schema-Tool
+```
+and then running:
+```bash
+> bost --help
+Usage: bost [OPTIONS]
```

Comment:
einerseits mag ich, dass es so ausführlich ist, andererseits doppeln wir damit doch das, was bei `--help rauskommt`?

Pflegen wir diese Doku hier, wenn wir die Hilfstexte anpassen? Andernfalls vllt nur in einem Kurzen Satz auf --help verweisen.

Reply:
> doppeln wir damit doch das, was bei `--help rauskommt`?

Was meinst du mit doppeln? Weil ich darunter auf die meisten Optionen (aber nicht alle) nochmal eingehe? Mir gefällt das eigentlich ganz gut so. Einmal in komprimiert und einmal etwas ausführlicher.



> Pflegen wir diese Doku hier, wenn wir die Hilfstexte anpassen?

So der Plan.